views/layout.hbs

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <title>IronBeers</title>
+    <link rel="stylesheet" href="/stylesheets/styles.css" />
+  </head>
+  <body>
+    <nav class="navbar">
+        <a href="/">Home</a>
+        <a href="/beers">Beers</a>
+        <a href="/random-beer">Random Beer</a>
+    </nav>
+    {{{ body }}}
```

Comment:
You could add a footer here! 👍

Besides that, everything looks great. You can just make it look nicer and delete the commented code.